packages/support/resources/lang/no/components/pagination.php

```diff
@@ -10,7 +10,7 @@
 
         'records_per_page' => [
 
-            'label' => 'Per side',
+            'label' => 'Pr. side',
```

Comment:
`Per side` wasn't correct in Norwegian?
All translation sites do give me that when translating `Per page`.

Reply:
You can use "Per side", but since "Per" is a common norwegian male name "Pr." is often used. It's a readability thing.

Reply:
Ha, interesting!